specs/agents/tracing-api-otel.md

```diff
@@ -0,0 +1,128 @@
+## OpenTelemetry API (Tracing)
+
+[OpenTelemetry](https://opentelemetry.io) (OTel in short) provides a vendor-neutral API that allows to capture tracing, logs and metrics data.
+
+Agents may provide a bridge implementation of OpenTracing Tracing API following this specification.
```

Comment:
```suggestion
Agents may provide a bridge implementation of OpenTelemetry Tracing API following this specification.
```

or did you actually mean OpenTracing here? (If so, then I am a little confused.)